github-actions/branch-manager/lib/main.ts

```diff
@@ -61,36 +65,58 @@ async function run() {
       const hasMergeLabel = payload.pull_request.labels.some(
         ({name}) => name === actionLabels.ACTION_MERGE.name,
       );
-      if (hasMergeLabel) {
-        await createWorkflowForPullRequest();
+      if (!hasMergeLabel) {
```

Comment:
Should it update in both states then? 

It's also worth mentioning that the state is easily getting out of sync. E.g. consider:

1. PR opened -> I get an update on mergability. Great!
2. Pushes happening
3. My PR is not updated with mergabilty because it has no merge ready label!!

Instead, an option is to consistently check all PRs, even for push, that are more recent than e.g. last 3 months

Reply:
Switched it back to how it was, it was an errant change.